### PR TITLE
Extends zoom level

### PR DESCRIPTION
**Pull request by Xevib**
I added a pseudo-dynamic zoom level support for Carto because we need more zoom levels

### CODE REVIEW

File: lib/carto/tree/zoom.js [hunk 1]
```diff
@@ -87,7 +87,10 @@ tree.Zoom.ranges = {
     20: 750,
     21: 500,
     22: 250,
-    23: 100
+    23: 100,
+    24: 50,
+    25: 25,
+    26: 13
```
Comment (tmcw) [inline lib/carto/tree/zoom.js]: why isn't this 12.5?

**Comment by Xevib** [comment]
@tmcw Watching the rest I suposed that floats wasn't allowed , it can be 12.5 if you want

**Comment by tmcw** [comment]
Yes, the value should be 12.5.

**Comment by tmcw** [comment]
:+1:

**Pull request merged**